### PR TITLE
the giant indentation patch of 2016, for great glory

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
Same connection errors. This passes green on my own travis account https://travis-ci.org/Br3nda/growstuff/builds/178962255

**Comment by Br3nda** [comment]
This is easier to review, if you look at one commit at a time.

I've gone with rubocop defaults, except on the indentation of multiple method calls.

I greatly prefer going with the rubocop defaults unless there's something special about a project. I work on a dozen or more rails app on any given week, i don't have the braincells to devote to remembering any special codestyle for one project..

**Comment by Br3nda** [comment]
Super keen to get this merged, and not resolve git conflicts for days. :hamburger:

**Comment by maco** [comment]
which means I should stop playing whack-a-mole on other issues and thus
creating merge conflicts

**Comment by Br3nda** [comment]
Me too. I have a bunch of whack-a-mole fixes to submit too :)

**Comment by pozorvlak** [comment]
@Br3nda: good point - I can definitely see the sense in sticking with rubocop's defaults, even if they're not my personal preference. But AFAICT that's a fairly small part of the PR, so let's get this merged quickly as-is to prevent spiralling merge conflicts of doom, and then argue about the points of contention in yet another PR?

File: spec/factories/comments.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@
     post
     author
     sequence(:body) { |n| "OMG LOL #{n}" } # because our commenters are more
```
Comment (pozorvlak) [inline spec/factories/comments.rb]: This is now kinda horrible - I'll fix it in a followup patch.

**Comment by pozorvlak** [comment]
On second thoughts, "everything gets indented level with the first parameter, except method calls" doesn't make a lot of sense. And "go with the community defaults" makes sense given our aim of being an approachable first project for new Rails developers. So I withdraw my objection to level-with-first-param indentation everywhere, even though I personally hate it.

[Pet peeve: PEP 8 [allows both styles](https://www.python.org/dev/peps/pep-0008/#indentation), allowing bikeshedding to continue.]

**Comment by pozorvlak** [comment]
Let's switch to rubocop's whitespace-tastic method-call defaults in a followup PR :cry:

File: app/controllers/registrations_controller.rb [hunk 2]
```diff
@@ -8,23 +8,22 @@ def edit
...
+  # https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password
 
   def update
-
     @member = Member.find(current_member.id)
 
-    successfully_updated = if needs_password?(@member, params)
-      @member.update_with_password(devise_parameter_sanitizer.sanitize(:account_update))
+    if needs_password?(@member, params)
+      successfully_updated = @member.update_with_password(devise_parameter_sanitizer.sanitize(:account_update))
```
Comment (Br3nda) [inline app/controllers/registrations_controller.rb]: It does, but here's an ignore for that file.

another PR that fixes yet more coming soon.

**Pull request merged**